packages/bpk-component-card/src/BpkDividedCard.js

```diff
@@ -52,6 +54,7 @@ const BpkDividedCard = (props: Props) => {
   const classNames = getClassName(
     'bpk-divided-card',
     isVertical ? 'bpk-divided-card--vertical' : 'bpk-divided-card--horizontal',
+    !isElevated && 'bpk-divided-card--depressed',
```

Comment:
The naming `depressed` might be confusing here. I suggest `bpk-divided-card--no-elevation` as it's consistent with the rest of the code and with the naming of the prop `isElevated`?

Reply:
Agree on taht, have renamed it to `--no-elevation` and `NonElevatedDividedCard`.